### ISSUE TITLE
Removal of libidn2 broke yum

### ISSUE THREAD

**Issue by m-witt** [plesk/centos2alma#367]
**Describe the bug**
When trying to update the following error occured 

```
2024-10-08 22:23:11,633 - ERROR - centos2alma process has failed. Error: Failed: updating mariadb databases. The reason: Command '['/usr/bin/package-cleanup', '--dupes']' returned non-zero exit status 1.
```

```
[root@XXXX ~] # /usr/bin/package-cleanup --dupes
Traceback (most recent call last):
  File "/usr/bin/package-cleanup", line 27, in <module>
    from yum.misc import setup_locale
  File "/usr/lib/python2.7/site-packages/yum/__init__.py", line 59, in <module>
    import config
  File "/usr/lib/python2.7/site-packages/yum/config.py", line 30, in <module>
    from parser import ConfigPreProcessor, varReplace
  File "/usr/lib/python2.7/site-packages/yum/parser.py", line 4, in <module>
    import urlgrabber
  File "/usr/lib/python2.7/site-packages/urlgrabber/__init__.py", line 55, in <module>
    from grabber import urlgrab, urlopen, urlread
  File "/usr/lib/python2.7/site-packages/urlgrabber/grabber.py", line 550, in <module>
    import pycurl
ImportError: libidn2.so.0: cannot open shared object file: No such file or directory
```

Seems like libidn2 is removed but not reinstalled before mariadb update is tried : 
```
2024-10-08 22:23:09,332 - INFO - Running: ['/usr/bin/rpm', '-e', '--nodeps', 'galera', 'python36-lxml', 'python36-six', 'libpcap', 'libidn2', 'lua-socket']. Output:
2024-10-08 22:23:09,727 - INFO - Command ['/usr/bin/rpm', '-e', '--nodeps', 'galera', 'python36-lxml', 'python36-six', 'libpcap', 'libidn2', 'lua-socket'] finished successfully
```
Tried 1.4.0, 1.4.1, 1.4.2

**Feedback archive**
[centos2alma_feedback.zip](https://github.com/user-attachments/files/17305508/centos2alma_feedback.zip)

**Comment by m-witt** [comment]
Seems to be related to [this](https://github.com/plesk/centos2alma/tree/nvizovitin-dont-remove-libidn2) Branch

**Comment by SandakovMM** [comment]
Yes, the change will be available in the next release. I plan to publish it by October 25th

**Comment by SandakovMM** [comment]
The problem was fixed in #358
